song-server/src/main/resources/schemas/sequencingRead.json

```diff
@@ -157,8 +157,9 @@
         }
       }
     },
-    "files": {
```

Comment:
why?

Reply:
This is a breaking change, and this is an array field anyways.

Reply:
it seems like a silent typo to me, since "files" is not in the required list and its also not a member in Analysis.java (https://github.com/icgc-dcc/SONG/blob/develop/song-server/src/main/java/org/icgc/dcc/song/server/model/analysis/Analysis.java#L36). 

If you compare to variantCall.json, "file" is used instead for listing the files.

Reply:
confirmed with @KevinFHartmann, this is a typo